content/versions-comparison/2.0-news.json

```
@@ -1,6 +1,6 @@
 [
   {
-    "title":"Product models and variants modeling",
+    "title":"New modeling with product with variants",
```

The two "with" sounds weird.
```suggestion
    "title":"New modeling for products with variants",
```